### DISCUSSION TITLE
`-skip_io`

### DISCUSSION THREAD

**Discussion by kareefardi** [The-OpenROAD-Project/OpenROAD#3962] [Q&A] [answered]
`-skip_io` in global_placement is not documented. What does it do and what are its uses?

**Comment by maliberty** [comment, marked as answer]
It ignore the IO ports when computing wirelength during placement.  There is a circular dependency between global placement and io placement.  We start the loop in ORFS by ignoring io placement while doing global placement, then do io placement, then re-run global placement considering IOs.  Its not perfect but that's the use.

@vvbandeira @luarss missing doc
